.devcontainer/.gitpod.Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM gitpod/workspace-full-vnc:latest
```

Comment:
A separate .gitpod directory would be fine with me, though I don't understand what you mean by its not being related to dev containers (I think Gitpod is a form of Docker-as-a-service, right?).  If we were ever to integrate the Gitpod support, one thing I would hope to do would be to figure out how to factor out the commonality between the Gitpod and local/VSCode Dockerfiles so that we didn't always have to, for example, add new packages to both (there is significant duplication between the `.gitpod.Dockerfile` in this PR and the existing `Dockerfile`, but I didn't find an obvious standard way of code-sharing between Dockerfiles, and didn't want to propose disturbing the existing `Dockerfile` for a speculative project at first anyway).